osgar/drivers/oak_camera.py

```diff
@@ -44,9 +57,9 @@ def __init__(self, config, bus):
 
         compress_depth_stream = config.get('compress_depth_stream', True)
         self.bus.register('depth:gz' if compress_depth_stream else 'depth',
-                          'color', 'orientation_list', 'detections',
+                          'color', 'orientation_list', 'detections', 'left_im', 'right_im',
```

Comment:
p.s. I am still thinking about `left_im` if `left_img` would be better ... but then you have `left_im_seq` ... probably ignore :(